<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1617</width>
    <height>1023</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTipDuration">
   <number>-32</number>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MplWidget" name="MplWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>1611</width>
      <height>771</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>471</width>
      <height>192</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>6</number>
     </property>
     <item row="1" column="3">
      <widget class="QSpinBox" name="spinBoxWindowSize">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
       <property name="value">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBoxDir"/>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxMf">
       <property name="minimum">
        <double>-1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>2.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButtonGenHeatmap">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Generate
Heatmap</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxQMin">
       <property name="minimum">
        <double>0.050000000000000</double>
       </property>
       <property name="maximum">
        <double>3.200000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.400000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBoxFile"/>
     </item>
     <item row="2" column="0">
      <widget class="QSlider" name="horizontalSliderFrame">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxQMax">
       <property name="minimum">
        <double>0.060000000000000</double>
       </property>
       <property name="maximum">
        <double>3.180000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>3.180000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxTransparency">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.250000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidgetFrames">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>1</y>
      <width>91</width>
      <height>191</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDelFrame">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClearFrame">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>0</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFixed">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTransparent">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetFrames">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>30</y>
      <width>661</width>
      <height>161</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxTransQValue">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.050000000000000</double>
    </property>
    <property name="maximum">
     <double>3.200000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.340000000000000</double>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTransTemplate">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxFixedTemplate">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBoxFixedQValue">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.050000000000000</double>
    </property>
    <property name="maximum">
     <double>3.200000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.070000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDimQPoints">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>130</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>51</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDimStartQPoints">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxHistBin">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>160</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1617</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
